docs/content/docs/tutorials/aurora-serverless-v2.md

```diff
@@ -0,0 +1,337 @@
+```
+
+You can now use the Helm chart to deploy the ACK service controller for
+Amazon RDS to your Amazon EKS cluster. At a minimum, you need to specify the
+AWS Region to execute the RDS API calls.
+
+```bash
+helm install --create-namespace \
+  --namespace "${ACK_SYSTEM_NAMESPACE}"  \
+  --set aws.region="$AWS_REGION" \
+  "ack-${SERVICE}-controller" "${CHART_EXPORT_PATH}/${SERVICE}-chart"
+
+helm list --namespace "${ACK_SYSTEM_NAMESPACE}" -o yaml    
+```
+
+### Configure IAM permissions
+
+Once the service controller is deployed, you will need to [configure the IAM permissions][irsa-permissions] for the controller to query the RDS API. For full details, please review the AWS Controllers for Kubernetes documentation for [how to configure the IAM permissions][irsa-permissions]. If you follow the examples in the documentation, use the value of `rds` for `SERVICE`.
+
+## Create an Aurora Serverless v2 PostgreSQL database
+
+To create an Aurora Serverless v2 database using the PostgreSQL engine, you must
+first create a DBSubnetGroup and a SecurityGroup for the VPC:
+
+```bash
+EKS_VPC_ID=$(aws eks describe-cluster --name "${EKS_CLUSTER_NAME}" --query "cluster.resourcesVpcConfig.vpcId" --output text)
+
+RDS_SUBNET_GROUP_NAME="mydbsubnetgroupack"
+RDS_SUBNET_GROUP_DESCRIPTION="mydb-subnetgroup"
+EKS_SUBNET_IDS=$(aws ec2 describe-subnets --filter "Name=vpc-id,Values=${EKS_VPC_ID}" --query 'Subnets[?MapPublicIpOnLaunch==`false`].SubnetId' --output text)
+
+cat <<-EOF > db-subnet-groups.yaml
+apiVersion: rds.services.k8s.aws/v1alpha1
+kind: DBSubnetGroup
+metadata:
+ name: ${RDS_SUBNET_GROUP_NAME}
+ namespace: ${APP_NAMESPACE}
+spec:
+ name: ${RDS_SUBNET_GROUP_NAME}
+ description: ${RDS_SUBNET_GROUP_DESCRIPTION}
+ subnetIDs:
+$(printf " - %s\n" ${EKS_SUBNET_IDS})
+ tags: []
+EOF
+
+kubectl apply -f db-subnet-groups.yaml
+
+RDS_SECURITY_GROUP_NAME="ACK_SG"
+RDS_SECURITY_GROUP_DESCRIPTION="ACK security group"
+
+EKS_CIDR_RANGE=$(aws ec2 describe-vpcs \
+ --vpc-ids "${EKS_VPC_ID}" \
+ --query "Vpcs[].CidrBlock" \
+ --output text
+)
+
+RDS_SECURITY_GROUP_ID=$(aws ec2 create-security-group \
+ --group-name "${RDS_SUBNET_GROUP_NAME}" \
+ --description "${RDS_SUBNET_GROUP_DESCRIPTION}" \
+ --vpc-id "${EKS_VPC_ID}" \
+ --output text
+)
+aws ec2 authorize-security-group-ingress \
+ --group-id "${RDS_SECURITY_GROUP_ID}" \
+ --protocol tcp \
+ --port 5432 \
+ --cidr "${EKS_CIDR_RANGE}"
+```
+
+Set up a master password using a Kubernetes Secret. SET `RDS_DB_USERNAME` and `RDS_DB_PASSWORD` to your preferred values for your RDS credentials:
+
+```bash
+export APP_NAMESPACE=mydb
+kubectl create ns "${APP_NAMESPACE}"
+
+RDS_DB_USERNAME="adminer"
+RDS_DB_PASSWORD="password"
+
+kubectl create secret generic -n "${APP_NAMESPACE}" ack-creds \
+  --from-literal=username="${RDS_DB_USERNAME}" \
+  --from-literal=password="${RDS_DB_PASSWORD}"
+```
+
+You can now create an Aurora Serverless v2 cluster:
```

Comment:
can we mention that for serverlessV2ScalingConfiguration, both maxCapacity and minCapacity need appear. it's required by rds api.

Reply:
:+1: